src/components/Tooltip/Tooltip.vue

```diff
@@ -17,16 +17,16 @@
           :class="[
             'absolute bg-transparent w-0 h-0 m-auto',
             {
-              'border-l-8 border-r-8 border-b-8 border-l-transparent border-r-transparent border-b-primary-700 -top-2': hasUpArrow
+              'border-l-8 border-r-8 border-b-8 border-l-transparent border-r-transparent border-b-gray-900 -top-2': hasUpArrow
```

Comment:
we have an opportunity for improvement here!
see how `border-l-8 border-r-8 border-b-8 border-l-transparent border-r-transparent border-b-gray-900` is repeated for all 4 arrow cases?
ideally we'd only write this once like:
`'border-l-8 border-r-8 border-b-8 border-l-transparent border-r-transparent border-b-gray-900' : hasArrow`
then add the position only per individual `hasXArrow` case
what do you think? happy to pair on it

Reply:
Yeah! Let's pair after all the meetings today

Reply:
ignore me they are all slightly different! 🤦🏼‍♀️ no refactoring needed